charts/greptimedb-cluster/values.yaml

```diff
@@ -1308,6 +1308,21 @@ slowQuery:
   # -- The TTL of slow query system table.
   ttl: "30d"
 
+## -- The license configuration for enterprise features.
+license:
+  # -- Enable enterprise features.
+  enabled: false
+  # -- The secret name to store the license. If not set, it will use the ${release-name}-license.
```

Comment:
The note of `secretName` is incorrect, always `license-secret` for now: https://github.com/GreptimeTeam/helm-charts/pull/353/files#diff-f4a14d6f635e86dd51a33622825fe489665ae004439ef27cec964698f823495cR356